fplus-lib/src/core/mod.rs

```diff
@@ -2501,47 +2501,6 @@ impl LDNApplication {
                 log::info!("Validated!");
                 return Ok(true);
             }
-
-            let app_file = application_file.move_back_to_governance_review();
-            let ldn_application =
-                LDNApplication::load(app_file.id.clone(), owner.clone(), repo.clone()).await?;
-            let parsed_app_file = serde_json::to_string_pretty(&app_file)
-                .map_err(|e| LDNError::Load(format!("Failed to pare into string: {e}")))?;
-            let new_file_sha = LDNPullRequest::add_commit_to(
-                ldn_application.file_name.clone(),
-                ldn_application.branch_name.clone(),
-                "Move application back to review".to_string(),
-                parsed_app_file.clone(),
-                ldn_application.file_sha.clone(),
-                owner.clone(),
-                repo.clone(),
-            )
-            .await?;
-
-            let gh = github_async_new(owner.to_string(), repo.to_string()).await?;
-
-            let prs = gh
-                .get_pull_request_by_head(&ldn_application.branch_name)
-                .await
-                .map_err(|e| LDNError::Load(format!("Failed to get pull request by head: {e}")))?;
-
-            if let Some(pr) = prs.first() {
-                let number = pr.number;
-                database::applications::update_application(
-                    app_file.id.clone(),
-                    owner,
-                    repo,
-                    number,
-                    parsed_app_file,
-                    Some(ldn_application.file_name.clone()),
-                    new_file_sha,
-                    app_file.client_contract_address,
-                )
-                .await
-                .map_err(|e| LDNError::Load(format!("Failed to update application: {e}")))?;
-            };
-
-            return Ok(false);
```

Comment:
This return should probably stay, otherwise we'll go do `log::info! failed to fetch application file` which definitely isn't something that we want.